src/SWP/Bundle/MultiTenancyBundle/spec/DependencyInjection/Compiler/RegisterOrganizationFactoryPassSpec.php

```diff
@@ -0,0 +1,111 @@
+<?php
```

Comment:
that's the whole idea of the phpspec tool - you should first write specs describing the behaviour before the implementation.